src/content/docs/nrql/nrql-tutorials/nrql-tutorial-process-your-data.mdx

```diff
@@ -561,7 +561,7 @@ SINCE 1 week ago
     </Step>
 </Steps>
 
-Digging into your data is always fun! Time-based cohorting is an easy way to expose problems that occur on specific minutes, hours, days, or weeks. No matter what data you send to New Relic, NRQL allows you to slice, dice, organize, and visualize it however you want.
+Hopefully you can now see how time-based cohorting exposes problems that occur on specific minutes, hours, days, or weeks. No matter what data you send to New Relic, NRQL allows you to slice, dice, organize, and visualize it however you want.
```

Comment:
Best to avoid "hopefully"

Reply:
Unless it is about me hopefully remembering not to use "hopefully" in the future! (fixed)